### ISSUE TITLE
JSON Decode error on doing keep init

### ISSUE THREAD

**Issue by nitinkgp23** [OrkoHunter/keep#41]
This is the full traceback of the error I am getting:

```
anomaly_@fibonacci ~/Downloads/keep (master) $ keep init
[CRITICAL] Remove everything inside ~/.keep ? [y/N]: y
Initializing environment in ~/.keep directory
...OK
Proceed to register? [Y/n]: y
Your credentials will be saved in the ~/.keep directory.
Email: nitch13jan@gmail.com
Repeat for confirmation: nitch13jan@gmail.com
Verifying with existing users...
Traceback (most recent call last):
  File "/home/anomaly_/miniconda3/bin/keep", line 11, in <module>
    load_entry_point('keep==2.4.2', 'console_scripts', 'keep')()
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/keep-2.4.2-py3.6.egg/keep/commands/cmd_init.py", line 14, in cli
    utils.first_time_use(ctx)
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/keep-2.4.2-py3.6.egg/keep/utils.py", line 54, in first_time_use
    register()
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/keep-2.4.2-py3.6.egg/keep/utils.py", line 125, in register
    if r.json()['exists']:
  File "/home/anomaly_/miniconda3/lib/python3.6/site-packages/requests/models.py", line 892, in json
    return complexjson.loads(self.text, **kwargs)
  File "/home/anomaly_/miniconda3/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/home/anomaly_/miniconda3/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/anomaly_/miniconda3/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

I have tried building the package from source too, but the same error comes.

**Comment by OrkoHunter** [comment]
This was because the herokuapp had crashed and returned something unexpected in response.

**Comment by hkinks** [comment]
Still getting the same error. :( 
```
$> keep update
Keep is at its latest version v2.4.2
```

**Comment by ldgregory** [comment]
I'm having the exact same error on Ubuntu 18.04. I can't get it to init or register. Both return the same error. Email address removed below for privacy reasons.

~$ keep register
Proceed to register? [Y/n]: Y
Your credentials will be saved in the ~/.keep directory.
Email: name@domain.com
Repeat for confirmation: name@domain.com
Verifying with existing users...
Traceback (most recent call last):
  File "/usr/local/bin/keep", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/keep/commands/cmd_register.py", line 13, in cli
    utils.register()
  File "/usr/local/lib/python2.7/dist-packages/keep/utils.py", line 124, in register
    if r.json()['exists']:
  File "/usr/local/lib/python2.7/dist-packages/requests/models.py", line 896, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded

**Comment by OrkoHunter** [comment]
Hello @ldgregory, Thanks for reminding me about it again! :D 

I am actually planning to use user's private GitHub Gist instead of a herokuapp. I plan to fix this soon!

**Comment by ldgregory** [comment]
> I am actually planning to use user's private GitHub Gist instead of a herokuapp. I plan to fix this soon!

No worries. I'm enjoying keep as is for now. Looking forward to being able to sync across devices once you've got it switched over to Github Gist though.

Thanks.

**Comment by pjmartorell** [comment]
I have the same problem but for the moment I don't to sync my commands.

**Comment by halloleo** [comment]
Got the same problem:

```
  File "/usr/local/Cellar/python/3.7.1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by OrkoHunter** [comment]
Fixed in #46 

Please update Keep to use
```
pip3 --no-cache-dir install -U keep
```

The latest version as of now is 2.5.2
